src/scenes/Users/List/UserSortOptions.tsx

```diff
@@ -0,0 +1,27 @@
+import { ComponentType } from 'react';
+import * as React from 'react';
+import { User } from '../../../api';
+import { SortOption, SortOptionProps } from '../../../components/Sort';
+
+// Alias component to define generic once
+const Option = SortOption as ComponentType<SortOptionProps<User>>;
+
+export const UserSortOptions = () => (
+  <>
+    <Option
+      default
+      value="displayFirstName"
```

Comment:
Yeah this is tricky since there is real name and display name. We don't want to sort based on real name if they don't have access to read that name. @michaelmarshall What do you think about sorting user's based on 2 of the 4 name properties?

Reply:
@CarsonF @michaelmarshall Did we ever get a ruling on this?